cnf-certification-test/platform/isredhat/isredhat.go

```diff
@@ -33,14 +32,12 @@ const (
 )
 
 type BaseImageInfo struct {
-	timeout      time.Duration
-	ClientHolder *clientsholder.ClientsHolder
+	ClientHolder clientsholder.Command
```

Comment:
ClientHolder seems a bit misleading name for this field, as now it's just the Command interface. What about:
```
CmdExecutor clientsholder.Command
```
Also, the clientholder's "Command" interface name is not following the [go-lang naming convention for interfaces](https://go.dev/doc/effective_go#:~:text=By%20convention%2C%20one%2Dmethod%20interfaces,the%20function%20names%20they%20capture.). I believe we should come up with a better name for that interface like "Commander" or "CmdExecutor", but it can be done in further PRs.